scheme/amd-kds-coserv/coserv_handler.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package amdkdscoserv
+
+import (
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/corim/coserv"
+)
+
+type CoservProxyHandler struct{}
+
+var (
+	dummyAuthority = []byte{0xab, 0xcd, 0xef}
+)
+
+func constructVcekUrl(instance *coserv.StatefulInstance) string {
+	// TODO(paulhowardarm) - deduce the product name and TCB parameters from the
```

Comment:
It would be really nice to have a github issue for Line 25 and Line 71 to track missing bits of requirement so that future work can be scoped out!